README.md

```diff
@@ -2,21 +2,37 @@
 
 # dotnet-jack
 
-A modern property-based testing tool, in the spirit of John Hughes & Koen Classen's [QuickCheck](https://web.archive.org/web/20160319204559/http://www.cs.tufts.edu/~nr/cs257/archive/john-hughes/quick.pdf). The key improvement is that shrinking is baked into the `Gen` monad, so you get it for free.
+A modern property-based testing tool, in the spirit of John Hughes & Koen Classen's [QuickCheck](https://web.archive.org/web/20160319204559/http://www.cs.tufts.edu/~nr/cs257/archive/john-hughes/quick.pdf). The key improvement is that shrinking comes for free.
 
 ![](https://github.com/moodmosaic/dotnet-jack/raw/master/img/dice.jpg)
 
 *Jack's love of dice has brought him here, where he has taken on the form of an F# library, in order to help you gamble with your properties.*
 
 ## Highlights
 
-* TODO
-* TODO
-* TODO
+* Shrinking is baked into the `Gen` type, so you get it for free.
+* Simplified model; just generators and properties.
+* Adequate randomness based on the SplitMix algorithm.
+* Convenient syntax for both generators and properties with **not only** `gen` but also `property` expressions.
 
 ## At a glance
 
-`dotnet-jack` is a lightweight, but powerful, property-based testing tool. The key improvement being that shrinking is baked in to the `Gen` monad, so you get it for free.
+`dotnet-jack` is a lightweight, but powerful, property-based testing tool. The key improvement being that shrinking is baked in to the `Gen` type, so you get it for free.
+
+```f#
+property {
+    let! x = Gen.range 1 100
+    let! ys = Gen.item [ "a"; "b"; "c"; "d" ] |> Gen.seq
+    counterexample (sprintf "tryHead ys = %A" <| Seq.tryHead ys)
+    return x < 25 || Seq.length ys <= 3 || Seq.contains "a" ys
```

Comment:
This is good to start, but perhaps we should try to find a more convincing example which demonstrates the same feature set?